wire-ios-data-model/Source/Model/Conversation/AssetCollectionBatched.swift

```diff
@@ -78,22 +78,24 @@ public class AssetCollectionBatched: NSObject, ZMCollection {
             fatal("syncMOC not accessible")
         }
         syncMOC.performGroupedBlock { [weak self] in
+            // swiftlint:disable redundant_self_in_closure
             guard let self, !self.tornDown else { return }
             guard let conversation = self.conversation,
                   let syncConversation = (try? syncMOC.existingObject(with: conversation.objectID)) as? ZMConversation else {
                 return
             }
-            let allAssetMessages: [ZMAssetClientMessage] = self.unCategorizedMessages(for: syncConversation)
-            let allClientMessages: [ZMClientMessage] = self.unCategorizedMessages(for: syncConversation)
+            // swiftlint:enable redundant_self_in_closure
```

Comment:
These comments are not needed and even create a `superfluous_disable_command` warning.